code/modules/chemistry/tools/food_and_drink.dm

```diff
@@ -309,6 +309,38 @@
 	afterattack(obj/target, mob/user , flag)
 		return
 
+	get_desc(mob/user)
+		if(!usr.traitHolder?.hasTrait("training_chef"))
+			return
+
+		if(src.quality >= 5)
+			. += "<br><span class='notice'>This is of great quality! The gained buffs will last longer! </span>"
+
+		if(length(food_effects) > 0)
+			. += "<br><span class='notice'> This food has the following effects: "
+			for(var/id as anything in src.food_effects)
+				var/datum/statusEffect/S = getStatusPrototype(id)
+				if(isnull(S))
+					logTheThing("debug", null, null, "the foodstuff [src] returned with a statusEffect ID that does not exist in the global prototype list! status_id : [id]") //This really shouldnt happen except for var editing, typos or other wierdness, but this is here just in case.
+					continue
+				var/Sdesc = S.getChefHint()
+				. += "<a href='byond://?src=\ref[src];action=chefhint;name=[url_encode(S.name)];txt=[url_encode(Sdesc)]'>[S.name]</a>" + "; "
```

Comment:
Hm, this is ok but I think that something as simple as just outputting a fixed string to a chat could be done without routing through Topic. If you want to look into that try peeking at browserOutput.js and seeing if you can do a function call of whatever the output function is called instead.

Reply:
Didnt end up working, RIP